docs/website/content/v0.3.en.json

```diff
@@ -99,6 +99,10 @@
         "title": "machined",
         "path": "v0.3/en/components/machined"
       },
+      {
+        "title": "apid",
+        "path": "v0.3/en/components/apid"
+      },
```

Comment:
I think what we went for in a previous PR is to keep these in alphabetical order.